app/views/miq_request/_request_dialog_details.html.haml

```diff
@@ -28,13 +28,14 @@
       = h(field.values.detect { |k, _v| k == wf.value(field.name) }.try(:last) || wf.value(field.name))
 
     - when "DialogFieldDropDownList"
-      = h(field.value.blank? ? _("<None>") : field.value.to_s.gsub("\x1F", ", "))
+      - if field.value.instance_of?(Array)
+        = h(field.value.blank? ? _("<None>") : h(field.value.join(", ")))
+      - else
+        = h(field.value.blank? ? _("<None>") : h(field.value))
 
     - when 'DialogFieldTagControl'
-      - value = wf.value(field.name) || '' # it returns in format Clasification::id
-      - classifications = value.split(',').map do |c|
-        - classification = Classification.find_by(id: c.split('::').second)
-        - classification.description if classification
+      - value = wf.value(field.name) || ''
```

Comment:
@GilbertCherrie Do you have a test case where `wf.value` can be nil?  I'm wondering if this should be 

```suggestion
      - value = wf.value(field.name) || []
```

Reply:
@Fryguy This is how it looks with empty values. The dropdown shows <None> where as the tag component just shows no value. 
<img width="1160" alt="Screenshot 2024-07-09 at 11 12 09 AM" src="https://github.com/ManageIQ/manageiq-ui-classic/assets/32444791/0c8c7d04-878e-49e8-a155-ee0e1dcc08ba">